ui/src/admin/components/EmptyRow.js

```diff
@@ -1,12 +1,11 @@
 import React, {PropTypes} from 'react'
 
-const EmptyRow = ({tableName}) => (
+const EmptyRow = ({tableName}) =>
   <tr className="table-empty-state">
     <th colSpan="5">
       <p>You don't have any {tableName},<br />why not create one?</p>
```

Comment:
That's a pretty common component. If you haven't seen your console light up, and webpack and prettier both say it's okay, I tend to agree with the tooling until I'm told otherwise. Pretty sure it's just a bug in GitHub's JSX syntax highlighter.